app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -0,0 +1,56 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::DetailedGuide < PublishingApiPresenters::Edition
+  def links
+    extract_links([
+      :lead_organisations
+    ]).merge(
+      related_guides: related_guides,
+      related_mainstream: related_mainstream
+    )
+  end
+
+private
+
+  def document_format
+    "detailed_guide"
+  end
+
+  def details
+    super.merge(
+      body: body,
+      first_public_at: first_public_at,
+      change_history: item.change_history.as_json,
+      related_mainstream_content: related_mainstream
+    )
+  end
+
+  def body
+    Whitehall::GovspeakRenderer.new.govspeak_edition_to_html(item)
+  end
+
+  def first_public_at
+    if item.document.published?
+      item.first_public_at
+    else
+      item.document.created_at.iso8601
+    end
+  end
+
+  def related_guides
+    item.try(:related_detailed_guide_ids) || []
+  end
+
+  def related_mainstream
+    base_paths = []
+    base_paths.push(item.related_mainstream_base_path)
+    base_paths.push(item.additional_related_mainstream_base_path)
+
+    if base_paths.any? # NB: Is false for [nil, nil].
+      publishing_api = Whitehall.publishing_api_v2_client
+      publishing_api.lookup_content_ids(base_paths: base_paths).values
```

Comment:
Also probably want `values.compact` here for things that weren't in publishing-api